script.js

```diff
@@ -0,0 +1,52 @@
+import { taskForm, taskTitle, taskList, btnCreate } from './elements.js';
+
+let tasks = [];
+let taskId = 0;
+
+function openForm() {
+    taskForm.classList.add("show");
+}
+
+function closeForm() {
+    taskForm.classList.remove("show");
+}
+
+function addTask() {
+    const title = taskTitle.value;
+    if (title === "") {
+        alert("Please enter a task title.");
+        return;
+    }
```

Comment:
We should not use any DOM blocking events. instead this you can show a error message for now and at your complete UI task you can add a toast message for this.

And is this full fill the complete input validation, like suppose we add some script tag code to the input field. should the task be added?

Reply:
validation not implemented, bhai